PolinaChernyshova/HW11/task2.py

```diff
@@ -0,0 +1,37 @@
+class CheckNumberError(Exception):
+    def __init__(self, data):
+        self.data = data
+
+    def __str__(self):
+        return repr(self.data)
+
+
+def get_day_by_number():
+    try:
+        user_number = int(input("Enter the number of week day: "))
+        if user_number < 1 or user_number > 7:
+            raise CheckNumberError(
+                "Error: Your number cannot be less than 1 or greater than 7"
+            )
+        match user_number:
```

Comment:
good idea
look at this implementation
def check_day_week(day):
    day_week = {
        1: 'Monday',
        2: 'Tuesday',
        3: 'Wednesday',
        4: 'Thursday',
        5: 'Friday',
        6: 'Saturday',
        7: 'Sunday'
    }

    try:
        enter_day=int(day)
    except ValueError:
        print('You did not enter a number!Please try again.')
    else:
        print(day_week .get(enter_day, 'There is no such day of the week!Please try again.'))